From e35cf05dbf04c6acfe17ae64625249d96488b984 Mon Sep 17 00:00:00 2001
From: Michal Simek <monstr@monstr.eu>
Date: Wed, 5 Jan 2011 13:50:11 +0100
Subject: [PATCH] microblaze: Extend time for sending chars through mdm IP

mdm uart support uses uartlite Linux driver. There was scrambled
output because mdm wasn't able to read chars.
Extending time for reading solved this issue.

Signed-off-by: Michal Simek <monstr@monstr.eu>
---
 drivers/tty/serial/uartlite.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/tty/serial/uartlite.c b/drivers/tty/serial/uartlite.c
index 6cd4143..6b17563 100644
--- a/drivers/tty/serial/uartlite.c
+++ b/drivers/tty/serial/uartlite.c
@@ -366,7 +366,7 @@ static void ulite_console_wait_tx(struct uart_port *port)
 	u8 val;
 
 	/* Spin waiting for TX fifo to have space available */
-	for (i = 0; i < 100000; i++) {
+	for (i = 0; i < 10000000; i++) {
 		val = ioread32be(port->membase + ULITE_STATUS);
 		if ((val & ULITE_STATUS_TXFULL) == 0)
 			break;
-- 
1.7.0.4

